lerna-package-lock.json

```diff
@@ -26358,9 +26568,9 @@
 			"dev": true
 		},
 		"async": {
-			"version": "3.2.0",
-			"resolved": "https://registry.npmjs.org/async/-/async-3.2.0.tgz",
-			"integrity": "sha512-TR2mEZFVOj2pLStYxLht7TyfuRzaydfpxr3k9RpHIzMgw7A64dzsdqCxH1WJyQdoe8T10nDXd9wnEigmiuHIZw=="
+			"version": "3.2.1",
```

Comment:
Something seems to have gone wrong here, the lockfile should have this set to 3.2.2 or higher since the packages now declare a dependency of at least 3.2.2. Might need to revert the changes to this file and run `npm i` again?